src/common.c

```diff
@@ -1197,6 +1202,8 @@ ssize_t ofi_bsock_send(struct ofi_bsock *bsock, const void *buf, size_t *len)
 	} else {
 		ret = bsock->sockapi->send(bsock->sockapi, bsock->sock, buf, *len,
 					   MSG_NOSIGNAL, bsock);
+		if (ret == -FI_EINPROGRESS_URING)
+			return ret;
```

Comment:
If we move this check down to the ```if (ret < 0)``` check just below, it will handle this case and the zerocopy case if that's ever a possibility.

Reply:
Done, thanks